Automattic/wp-calypso
# Odyssey Widget: button to install Akismet

Related to #76446 

## Proposed Changes

Show `Install` button for the installation and activation of the Akismet plugin.

## Testing Instructions


* Build Jetpack if necessary: `jetpack build plugins/jetpack`.
* Run `STATS_PACKAGE_PATH=/path/to/jetpack/projects/packages/stats-admin yarn dev` under `calypso/apps/odyssey-stats`.
* Uninstall Akismet if installed
* Open `/wp-admin/index.php` on a site with Jetpack bleeding edge
* Ensure `Install` button shows in Akismet card
* Click the button
* Ensure Akismet is installed and activated
* If there's no key configured with Akismet
* Ensure a link is shown for user to manage their Akismet key

<img width="626" alt="image" src="https://user-images.githubusercontent.com/1425433/235565994-97d2afdb-374e-49d7-8182-28e5f066bd10.png">

<img width="487" alt="image" src="https://user-images.githubusercontent.com/1425433/235572730-95564668-7fa9-447b-943b-70e856f3cf4a.png">


<img width="429" alt="image" src="https://user-images.githubusercontent.com/1425433/235572441-9ce51fea-ea54-491b-a210-1859fab78839.png">


## Pre-merge Checklist
